pkgs/development/python-modules/devito/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv
+, lib
+, buildPythonPackage
+, fetchFromGitHub
+, anytree
+, pytest-cov
+, codecov
+, nbval
+, flake8
+, numpy
+, sympy
+, scipy
+, cached-property
+, psutil
+, py-cpuinfo
+, cgen
+, click
+, multidict
+, distributed
+, pytest-runner
+, pyrevolve
+, codepy
+}:
+
+buildPythonPackage rec {
+  pname = "devito";
+  # grabbing the commit that has NixOS compatibility fix
+  version = "4.6.2-bbad3b1";
```

Comment:
Then either fetch the commits as patch or use the unstable-format. Coming up with new version numbers is a no go.

Reply:
Hm, I gonna take some time searching for a way to do the former, but the later seems straightforward.

Reply:
I ended up applying what the PR did.